src/main/resources/application.properties

```diff
@@ -6,5 +6,10 @@ authentication.oauth.clientid=webclient
 authentication.oauth.secret=secret
 authentication.oauth.tokenValidityInSeconds=43200
 
+#facebook credentials
 spring.social.facebook.appId=388810664881573
```

Comment:
We cant commit this to code unfortunately. 
Anybody can use these secret keys unfortunately.

Reply:
Check this answer. Exactly what you need here: 

https://stackoverflow.com/a/35535138/2359861

Reply:
i will fix this one